css/styles.css

```diff
@@ -561,47 +574,63 @@ Contains email and socials
   }
   */
 
+  header {
+    height: 8vh;
+  }
 
-    .headline,
-    .about-myself {
-      margin: 0;
-    }
+  header #union {
+    display: none;
+  }
 
-    .spacing-default {
-      margin: 2% auto;
-    }
+  header #navbar {
+    display: flex;
+  }
 
-    .arrow {
-      margin-bottom: 0;
-    }
+  .about-myself {
+    margin: 0;
+  }
 
-    .recent-work {
-      width: 92%;
-    }
+  .headline {
+    margin-top: 5%;
+  }
 
-    .recent-work-text,
-    .about-myself-text {
-      margin: 0 auto 5% auto;
-    }
+  .header-bg {
+    background-image: url(../images/Header-llustration-desktop.png);
+  }
+
+  .spacing-default {
+    margin: 2% auto;
+  }
+
+  .arrow {  /* TO CENTER */
+    margin: 0% auto 20% auto;
+  }
+
+  .recent-work {
+    width: 92%;
+  }
+
+  .recent-work-text,
+  .about-myself-text {
+    margin: 0 auto 5% auto;
+  }
 
-    .recent-work-text img {
-      width: auto;
-      position: relative;
-      margin: 1% auto 0% auto;
-    }
+  .recent-work-text img {
+    width: auto;
+    position: relative;
+    margin: 1% auto 0% auto;
+  }
```

Comment:
- [ ] In the figma design there are suppose to be six project cards. Please make sure your design also includes six cards. 

Your design:

<img width="1499" alt="Screenshot 2023-03-31 at 9 25 46 AM" src="https://user-images.githubusercontent.com/44798044/229031084-39927540-a393-4ab3-ab02-d1277abf5ce0.png">

Figma design:

<img width="500" alt="Screenshot 2023-03-31 at 9 27 34 AM" src="https://user-images.githubusercontent.com/44798044/229031036-0765a456-0fbd-4e2e-b442-47f5d043034b.png">

Reply:
Done